### PR TITLE
[typo](docs)add the best practices for hive catalog with kerberos

### PR DESCRIPTION
**Pull request by wsjz**
## Proposed changes

add kerberos best practices for hive catalog.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wsjz** [comment]
run buildall

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 1]
```diff
@@ -479,3 +456,67 @@ Apache Ranger是一个用来在Hadoop平台上进行监控，启用服务，以
...
 
+## 使用 Kerberos 进行认证
+
+Kerberos是一种身份验证协议。它的设计目的是通过使用私钥加密技术为应用程序提供强身份验证。
+
+### 环境配置
+
+1. 当集群中的服务配置了Kerberos认证，配置Hive Catalog时需要获取它们的认证信息。
+
+`hadoop.kerberos.keytab`: 记录了认证所需的principal，Doris集群中的keytab必须是同一个。
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:467]: 缩进4个空格

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 2]
```diff
@@ -479,3 +456,67 @@ Apache Ranger是一个用来在Hadoop平台上进行监控，启用服务，以
...
+
+在所有的 `BE`、`FE` 节点下放置 `krb5.conf` 文件和 `keytab` 认证文件，`keytab` 认证文件路径和配置保持一致，`krb5.conf` 文件默认放置在 `/etc/krb5.conf` 路径。
+
+2. 当配置完成后，如在`FE`、`BE`日志中无法定位到问题，可以开启Kerberos调试。
+
+ - 在所有的 `FE`、`BE` 节点下，找到部署路径下的`conf/fe.conf`以及`conf/be.conf`。
+
+ - 找到配置文件后，在`JAVA_OPTS`变量中设置JVM参数`-Dsun.security.krb5.debug=true`开启Kerberos调试。
+
+ - `FE`节点的日志路径`log/fe.out`可查看FE Kerberos认证调试信息，`BE`节点的日志路径`log/be.out`可查看BE Kerberos认证调试信息。
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:485]: 增加一个到 FAQ的链接，比如：“相关错误解决方法课参阅 xxx”

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 3]
```diff
@@ -479,3 +456,67 @@ Apache Ranger是一个用来在Hadoop平台上进行监控，启用服务，以
...
+
+`hadoop.kerberos.principal`: Doris集群上找对应hostname的principal，如`doris/hostname@HADOOP.COM`，用`klist -kt`检查keytab。
+
+`yarn.resourcemanager.principal`: 到Yarn Resource Manager节点，从 `yarn-site.xml` 中获取，用`klist -kt`检查Yarn的keytab。
+
+`hive.metastore.kerberos.principal`: 到Hive元数据服务节点，从 `hive-site.xml` 中获取，用`klist -kt`检查Hive的keytab。
+
+`hadoop.security.authentication`: 开启Hadoop Kerberos认证。
+
+在所有的 `BE`、`FE` 节点下放置 `krb5.conf` 文件和 `keytab` 认证文件，`keytab` 认证文件路径和配置保持一致，`krb5.conf` 文件默认放置在 `/etc/krb5.conf` 路径。
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:477]: 增加一个说明：同时需确认JVM参数 `-Djava.security.krb5.conf` 和环境变量 `KRB5_CONFIG` 指向了正确的 `krb5.conf` 文件的路径

**Comment by wsjz** [comment]
run buildall

**Comment by wsjz** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**